spec/acceptance/grafana_plugin_spec.rb

```diff
@@ -95,5 +95,31 @@ class { 'grafana':
         end
       end
     end
+
+    context 'grafana plugins' do
+      it 'installs' do
+        pp = <<-EOS
+        class { 'grafana':
+          version => 'latest',
+        }
+
+        package { 'grafana-image-renderer':
+          provider => 'grafana',
+        }
+        EOS
+        # Run it twice and test for idempotency
+        apply_manifest(pp, catch_failures: true)
+        apply_manifest(pp, catch_changes: true)
+      end
+
+      describe package('grafana-image-renderer').provider(:grafana) do
```

Comment:
anybody experienced enough to write this test correctly? Already hat it like `describe package('grafana-image-renderer') do` but was also failing (probably because it doesn't set provider to grafana?)